inbm/dispatcher-agent/fpm-template/usr/share/dispatcher-agent/schedule_manifest_schema.xsd

```diff
@@ -68,7 +80,7 @@
   <!-- Root element type -->
   <xs:complexType name="ScheduleRequest">
     <xs:sequence>
-      <xs:element name="request_id" type="xs:string"/>
+      <xs:element name="request_id" type="ValidUuidFormat"/>
```

Comment:
need to document in proto files that request ID has to be in this format and is expected to be unique or will be rejected. probably should add checking of this format in mjunct and INBS as well so we don't get a bad request ID accepted higher up and then rejected by turtle creek